<?xml version="1.0" encoding="UTF-8" ?>

<launch>
	<node pkg='pointcloud_to_laserscan' type='pointcloud_to_laserscan_node' name='pointcloud_to_laserscan'>

		<remap from='cloud_in' to='/camera/depth/color/points'/>
		<remap from='scan' to='/AMR/cam/scanF'/>

		<param name="target_frame" value="camera_link"/>

		<param name="transform_tolerance" value="0.01"/>

		<param name="min_height" value="0"/>
		<param name="max_height" value="5"/>

		<param name="angle_min" value="-1.5708"/>
		<param name="angle_max" value="1.5708"/>
		<param name="angle_increment" value="0.0087"/>

		<param name="scan_time" value="0.333"/>
		<param name="range_min" value="0.3"/>
		<param name="range_max" value="3.0"/>
		<param name="use_inf" value="true"/>
	
	</node>

</launch>